src/schemas/generator/v1/index.ts

```diff
@@ -0,0 +1,22 @@
+import { z } from 'zod'
+import { TestRuleSchema } from './rules'
+import { TestDataSchema } from './testData'
+import { TestOptionsSchema } from './testOptions'
+
+export const GeneratorFileDataSchema = z.object({
+  version: z.literal('1.0'),
+  recordingPath: z.string(),
+  options: TestOptionsSchema,
+  testData: TestDataSchema,
+  rules: TestRuleSchema.array(),
+  allowlist: z.string().array(),
+  includeStaticAssets: z.boolean(),
+  scriptName: z.string().default('my-script.js'),
+})
+
+export type GeneratorSchema = z.infer<typeof GeneratorFileDataSchema>
+
+// TODO: Migrate generator to the next version
```

Comment:
`v1` is essentially the same as `v0` here, right? I don't understand what do we need the TODO comment for - is it for future non-breaking changes to `v1`? If so, I'd prefer a regular descriptive comment, so that TODOs are reserved for debt

Reply:
They are the same schema, but the implementation of the `migrate` function is incomplete because there is nothing to migrate to since `v1` is the latest version.

The `TODO` comment here indicates that the implementation needs to change when another version exists (to migrate from `v1` to `v2`).